build.py

```diff
@@ -1643,8 +1643,15 @@ def create_build_dockerfiles(
 ):
     if "base" in images:
         base_image = images["base"]
+        if target_platform() == "rhel":
+            print(
+                "warning: RHEL is not an officially supported target and you will probably experience errors attempting to build this container."
+            )
     elif target_platform() == "windows":
         base_image = "mcr.microsoft.com/dotnet/framework/sdk:4.8"
```

Comment:
that's interesting: 
https://github.com/triton-inference-server/server/blob/main/Dockerfile.win10.min#L29

Reply:
No it's just an observation.